src/test/ui/structs-enums/rec-align-u64.rs

```diff
@@ -40,7 +40,8 @@ struct Outer {
           target_os = "macos",
           target_os = "netbsd",
           target_os = "openbsd",
-          target_os = "solaris"))]
+          target_os = "solaris",
+	  target_os = "vxworks"))]
```

Comment:
The indentaion looks not right too. Pleas check if we are using 'tab' instead of spaces.